weather.py

```diff
@@ -70,11 +67,11 @@ def get_weather(woeid, unit, format, timeout=None):
     p = argparse.ArgumentParser()
     p.add_argument('woeid')
     p.add_argument('--format', metavar='F',
-                   default=u'{city}, {region}: {text}, {temp}\u00b0{unit_temperature}',
+                   default='{city}, {region}: {text}, {temp}\u00b0{unit_temperature}',
                    help="format string for output")
     p.add_argument('--position', metavar='P', type=int, default=-2,
                    help="position of output in JSON when wrapping i3status")
-    p.add_argument('--unit', metavar='U', default='f',
+    p.add_argument('--unit', metavar='', default='f',
```

Comment:
Guessing the `metavar='U'` get caught by a sed to remove Python 2 unicode literals.  Please add it back to preserve the short form `-U` option.